stdlib/public/Concurrency/TaskGroup.cpp

```diff
@@ -694,7 +730,7 @@ static void swift_taskGroup_wait_next_throwingImpl(
     OpaqueValue *resultPointer, SWIFT_ASYNC_CONTEXT AsyncContext *callerContext,
     TaskGroup *_group,
     ThrowingTaskFutureWaitContinuationFunction *resumeFunction,
-    AsyncContext *rawContext) {
+    AsyncContext *rawContext) SWIFT_OPTNONE {
```

Comment:
Is `SWIFT_OPTNONE` supposed to be here? If so, can we have a comment explaining it?

Reply:
Nope, I did it for easier debugging.

Reply:
I'll have to remember that trick.